cookbook/security/entity_provider.rst

```diff
@@ -583,6 +583,11 @@ The code below shows the implementation of the
         }
     }
 
+.. tip::
+
+   Don't forget to 
+   :ref: `add the repository class to the mapping definition of your entity <book-doctrine-custom-repository-classes>`.
```

Comment:
The whole block should be indented by four spaces. And can we maybe shorten the `ref` label a bit (maybe using "of your entity" out of it)? What do you think?